Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string firstName = PromptString("What is your first name?");
+
```

Comment:
A blank line after every line is not needed. It did make sense before you didn't use functions (to isolate a function-like block), but now your functions make it readable enough.

The only bit that I would separate is reads from bmi calculation.

Also, the contents of demo could go into a function of its' own and you're missing person 2